Update 2020-07-31-ji20a.md (#1)
mlresearch/v120: _posts/2020-07-31-ji20a.md

---
title: Feed-forward Neural Networks with Trainable Delay
abstract: In this paper we build a bridge between feed-forward neural networks and delayed dynamical systems. As an initial demonstration, we capture the car-following behavior of a connected automated vehicle that includes time delay by using both simulation data and experimental data. We construct a delayed feed-forward neural network (DFNN) and introduce a training algorithm in order to learn the delay. We demonstrate that this algorithm works well on the proposed structures.
layout: inproceedings
series: Proceedings of Machine Learning Research
issn: 2640-3498
id: ji20a
month: 0
tex_title: Feed-forward Neural Networks with Trainable Delay
firstpage: 127
lastpage: 136
page: 127-136
order: 127
cycles: false

> bibtex_author: Ji, Xunbi and Molnar, Tamas and Avedisov, Sergei and Orosz, Gabor

Reviewer: ```suggestion
bibtex_author: Ji, Xunbi and Moln\'ar, Tam\'as and Avedisov, Sergei and Orosz, Gabor
```

Author: Thank you! 
I thought it was the format to drop the accent and middle name...If we add the accent, we need one more accent in G\'abor. Also for the middle names, maybe it is good to have them too.
```suggestion
bibtex_author: Ji, Xunbi A. and Moln\'ar, Tam\'as G. and Avedisov, Sergei S. and Orosz, G\'abor
```

Reviewer: Thanks for adding Gabor's accent. 

Update with middle initials was perfect also. Thank you!

author:
- given: Xunbi
  family: Ji

> - given: Tamas

Reviewer: ```suggestion
- given: Tamás
```

>   family: Molnar

Reviewer: ```suggestion
  family: Molnár
```
